interfaces/user.ts

```diff
@@ -0,0 +1,13 @@
+import Joi from 'joi';
+
+const schema = Joi.object({
+  firstName: Joi.string().when('$strict', { is: true, then: Joi.required() }),
+  lastName: Joi.string().when('$strict', { is: true, then: Joi.required() }),
```

Comment:
let's capitalize the first name and last name before putting it into the db (might not be through Joi)